### PR TITLE
DBCP-576 - Simplify and inline variables

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: src/test/java/org/apache/commons/dbcp2/TestPoolingDataSource.java [hunk 1]
```diff
@@ -101,9 +101,8 @@ public void testPoolGuardConnectionWrapperEqualsSameDelegate() throws Exception
     public void testPoolGuardConnectionWrapperEqualsReflexive()
         throws Exception {
         final Connection con = ds.getConnection();
-        final Connection con2 = con;
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/dbcp2/TestPoolingDataSource.java:104]: Don't change there I think. See the title of the method. This makes it obvious.

File: src/test/java/org/apache/commons/dbcp2/managed/TestManagedDataSource.java [hunk 2]
```diff
@@ -198,9 +198,8 @@ public void testManagedConnectionEqualsSameDelegate() throws Exception {
     @Test
     public void testManagedConnectionEqualsReflexive() throws Exception {
         final Connection con = ds.getConnection();
-        final Connection con2 = con;
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/dbcp2/managed/TestManagedDataSource.java:201]: Don't change there I think. See the title of the method. This makes it obvious.

**Review by garydgregory — CHANGES_REQUESTED** [review]
See inline comments.

File: src/test/java/org/apache/commons/dbcp2/managed/TestManagedDataSource.java (hunk 2 above)
Comment (arturobernalg) [reply src/test/java/org/apache/commons/dbcp2/managed/TestManagedDataSource.java]: Done

File: src/test/java/org/apache/commons/dbcp2/TestPoolingDataSource.java (hunk 1 above)
Comment (arturobernalg) [reply src/test/java/org/apache/commons/dbcp2/TestPoolingDataSource.java]: Done

File: src/test/java/org/apache/commons/dbcp2/datasources/TestSharedPoolDataSource.java [hunk 3]
```diff
@@ -556,7 +554,7 @@ private void doTestPoolCallableStatements(final PrepareCallCallback callBack)
         conn.close();
         conn = null;
 
-        conn = myDs.getConnection();
+        conn = ((DataSource) spDs).getConnection();
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/dbcp2/datasources/TestSharedPoolDataSource.java:557]: This is not clearer than what was before. I would not do this.

Comment (arturobernalg) [reply src/test/java/org/apache/commons/dbcp2/datasources/TestSharedPoolDataSource.java]: HI @garydgregory 
reverted
TY

**Pull request merged**